src/controllers/articleController.js

```diff
@@ -92,6 +96,266 @@ class ArticleController {
       throw (err);
     }
   }
+
+  /**
+   * @description like an article
+   * @param {object} req
+   * @param {object} res
+   * @return {object} return object with a liked article
+   */
+  static async likeArticle(req, res) {
+    try {
+      const liker = req.userData.id;
+      const slugId = req.params.articleSlug;
+      const likeVote = 1;
+      const dislikeVote = 0;
+
+      const findArticle = await Article.findOne({ where: { slug: slugId } });
+
+      if (!findArticle) {
+        return res.status(200).json({
+          message: 'Article not found'
+        });
+      }
+
+      const reactedAlready = await Reaction.findOne({ where: { articleSlug: slugId, userId: liker } });
+
+      const { id } = findArticle;
+
+      if (!reactedAlready) {
+        const likedArticle = await Reaction.create({
+          articleId: id,
+          articleSlug: slugId,
+          userId: liker,
+          likes: likeVote,
+        });
+
+        const {
+          articleSlug, userId, likes
+        } = likedArticle;
+
+        if (likedArticle) {
+          await Article.increment({ likes: 1 }, { where: { slug: slugId } });
+
+          return res.status(200).json({
+            message: 'You have liked the article',
+            reaction: {
+              ArticleSLUG: articleSlug,
```

Comment:
fix the capitalization here, use camelCase

Reply:
Okay